source/common/http/conn_manager_impl.h

```diff
@@ -164,6 +165,19 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
     const Buffer::Instance* decodingBuffer() override {
       return parent_.buffered_request_data_.get();
     }
+    void sendLocalReply(Code code, const std::string& body,
+                        std::function<void(HeaderMap& headers)> modify_headers) override {
+      Utility::sendLocalReply(
+          is_grpc_request_,
+          [this, modify_headers](HeaderMapPtr&& headers, bool end_stream) -> void {
```

Comment:
Nit: Prefer not to have gratuitous `-> void`.

Reply:
Tried leaving the 'void' out and compiler (on OSX) says:
```
ERROR: /Users/jarno/dev/envoy/source/common/http/BUILD:92:1: C++ compilation of rule '//source/common/http:conn_manager_lib' failed (Exit 1)
In file included from source/common/http/conn_manager_utility.cc:1:
In file included from bazel-out/darwin-dbg/bin/source/common/http/_virtual_includes/conn_manager_lib/common/http/conn_manager_utility.h:9:
bazel-out/darwin-dbg/bin/source/common/http/_virtual_includes/conn_manager_lib/common/http/conn_manager_impl.h:172:78: error: expected a type
          [this, modify_headers](HeaderMapPtr&& headers, bool end_stream) -> {
                                                                             ^
bazel-out/darwin-dbg/bin/source/common/http/_virtual_includes/conn_manager_lib/common/http/conn_manager_impl.h:178:62: error: expected a type
          [this](Buffer::Instance& data, bool end_stream) -> { encodeData(data, end_stream); },
                                                             ^
2 errors generated.
```